src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -658,6 +853,12 @@ public void ReturnConnection(HttpConnection connection)
             }
         }
 
+        public void InvalidateHttp2Connection(Http2Connection connection)
```

Comment:
I didn't see, but maybe I missed it... are we respecting PooledConnectionLifetime for HTTP/2? i.e. do we cull connection after some period of time?  Or is there a TODO about respecting that somewhere?